src/components/Settings/ChangePasswordForm.js

```diff
@@ -29,26 +29,22 @@ const ChangePasswordForm = () => {
   };
 
   return (
-    <Form
-      formMethods={formMethods}
-      className={styles.settingsForm}
-      onSubmit={onSubmit}
-    >
+    <Form className={formClass} formMethods={formMethods} onSubmit={onSubmit}>
```

Comment:
Change `className` to `styles` and maybe `formClass` to `formStyles`. Same for the button bellow